tar/tarsnap.1-man.in

```diff
@@ -742,6 +742,11 @@ but it is likely that other
 warnings will be silenced by this option in future versions of
 \fB\%tarsnap\fP.
 .TP
+\fB\--resume-extract\fP
+(x mode only)
+Don't extract files whose filesize and mtime matches existing files on the
+disk.  Primarily used to resume an archive extraction which was interrupted.
+.TP
```

Comment:
Probably worth adding something along the lines of "This differs from -k in that --resume-extract will overwrite a file if the size or modification time do not match, as can happen if tarsnap is killed partway through extracting a file."